Added Runtime and Cloud release notes for 12/16 (#210)
astronomer/docs: docs/release-notes.md

If you have any questions or a bug to report, don't hesitate to reach out to [Astronomer Support](https://support.astronomer.io).

**Latest Runtime Version**: 4.0.7 ([Release notes](runtime-release-notes.md))

**Latest CLI Version**: 1.0.4 ([Release notes](cli-release-notes.md))

## December 16, 2021

### View Scheduler Error Logs from the Astronomer UI

> The new **Logs** tab shows Scheduler logs for all of your Deployments. When you select a Deployment in this menu, all error logs generated over the last 30 minutes appear in the UI.

Reviewer: Given we're filtering to a subset of Scheduler logs, we should have a modifier on `Scheduler logs`

Author: "Scheduler error and warning logs" is probably the most accurate we can get without getting in the weeds. Once we get some more ASAP changes to this feature, we can be more specific about the logs in a dedicated document.

> The new **Logs** tab shows Scheduler error and warning logs for all of your Deployments. When you select a Deployment in this menu, all error logs generated over the last 30 minutes appear in the UI.
> 
> ![Logs page in the UI](/img/release-notes/logs-page.png)
> 
> You can also directly access logs for a given Deployment by clicking the new **Logs** icon on the Deployment's page or in the **Deployments** table.
> 
> ![Logging direct access button](/img/release-notes/logs-button.png)

Reviewer 2: ```suggestion
The new **Logs** tab in the Astronomer UI shows Scheduler error and warning logs for all Deployments in your Workspace. When you select a Deployment in this menu, all error logs generated over the last 30 minutes appear in the UI.

![Logs page in the UI](/img/release-notes/logs-page.png)

To access logs directly for a given Deployment, click the new **Logs** icon on the Deployment's page or in the **Deployments** table.

![Logging direct access button](/img/release-notes/logs-button.png)
```

### Additional Improvements

> - Added new messages and handling for when non-admin users attempt to log in to an administrative page

Reviewer 2: @chomipi88 Is this for Management UI? I'm thinking we don't broadcast this to customers since it's very much not an expected path any customers should find themselves in?

Reviewer 3: Correct, let's remove this.

### Bug Fixes

> - Fixed various UI bugs that occurred when a Deployment had 0 metrics.  

Reviewer 2: ```suggestion
- Fixed various bugs in the Astronomer UI to better handle nulls and unknowns in Deployment metrics.
```
